### DISCUSSION TITLE
Allow "locking" of track selection and progress bar seeking

### DISCUSSION THREAD

**Discussion by fberg** [PaulWoitaschek/Voice#2195] [Ideas]
It would be great if Voice had an option to lock all playback related controls, apart from play/pause and the quick rewind/forward buttons. Ideally, this would also apply to the lock screen controls.

Context: I get most of my audiobook listening done while cycling, and when trying to rewind a few seconds, I often hit either the track selection drop-down (or the next/prev buttons) or a random location on the progress bar because the ride might be a bit bumpy. When this happens, I have to spend a few minutes to get back to where I was before. This happens more frequently again now that thick gloves are needed due to falling temperatures. :)

**Comment by ror3d** [comment]
Definitely, this is a big problem when an audiobook is a single, extremely long audio file, any kind of accidental touch on the progress bar will take a long time to get back to the right position having to seek through a 10+ hour long audio in a tiny progress bar.

**Comment by Soapsuds** [comment]
I also would love this feature! I might be interesting implementing this, but have pretty limited android skills.Has anyone with more experience looked into this? I'm wondering how difficult this would be and how one might go about doing it.
